.gitattributes

```diff
@@ -0,0 +1 @@
+src/styles/antd.* linguist-vendored
```

Comment:
This will fix this problem (CSS shows as the dominant language, should be TS or TSX)

<img width="326" alt="Screen Shot 2021-11-28 at 1 28 28 pm" src="https://user-images.githubusercontent.com/94939382/143728969-06722347-1294-4fb3-8b1c-93bb743d6c69.png">

Reply:
thanks!